src/Documentation/DocumentationGenerator/Extensions.cs

```diff
@@ -179,11 +179,37 @@ TypedExpression input
             // are an empty set; in that case, we want an empty string.
             ?? string.Empty;
 
-        internal static string ToSyntax(this QsCallable callable, Action<IRewriteStep.Diagnostic>? onDiagnostic = null) =>
-            SyntaxTreeToQsharp.DeclarationSignature(
+        internal static string ToSyntax(this QsFunctor functor) =>
+            functor.Tag switch
+            {
+                QsFunctor.Tags.Adjoint => "Adj",
+                QsFunctor.Tags.Controlled => "Ctl",
+                _ => "__invalid__"
+            };
+
+        internal static string ToSyntax(this QsCallable callable, Action<IRewriteStep.Diagnostic>? onDiagnostic = null)
+        {
+            var signature = SyntaxTreeToQsharp.DeclarationSignature(
                 callable, SyntaxTreeToQsharp.Default.ToCode
             );
```

Comment:
```suggestion
                callable, SyntaxTreeToQsharp.Default.ToCode);
```

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1111.md

Looks like there are a lot of StyleCop violations in this file, actually. :) Out of curiosity, are the squigglies showing up for you? It should look like:

![image](https://user-images.githubusercontent.com/33814365/98044005-0c84b880-1ddb-11eb-8ef9-20fe644fbbd4.png)

Reply:
I'll admit, I find code that follows SA1111 _much_ more difficult to read... maybe I've spent too long in PEP 8 codebases, which requires the exact opposite. In any case, thanks for raising that, will go on and fix.